react/Radio/index.jsx

```diff
@@ -15,6 +24,7 @@ const Radio = props => {
         className
       )}
       aria-disabled={disabled}
+      style={style}
```

Comment:
Is the `style` prop really necessary? Can we just use `...restProps` or something?

Reply:
restProps is already applied on the input below.